src/ape/managers/chain.py

```diff
@@ -482,7 +482,14 @@ def restore(self, snapshot_id: Optional[SnapshotID] = None):
         self.provider.revert(snapshot_id)
         self.account_history.revert_to_block(self.blocks.height)
 
-    def mine(self, num_blocks: int = 1, timestamp: Optional[int] = None) -> None:
+    def mine(
+        self,
+        num_blocks: int = 1,
+        timestamp: Optional[int] = None,
+        deltatime: Optional[int] = None,
+    ) -> None:
```

Comment:
Can we add a test or 2?

Reply:
I was waiting to get this to pass here, there was a connection issue so I had to run a few times before it did.